### ISSUE TITLE
The issue for optimizer

### ISSUE THREAD

**Issue by CHENJIAR3** [knu-plml/ecg-recon#3]
Dear authors, 
       We have reproduced your paper, and found it is difficult for us to reproduce your method. I have noticed that the used optimizer is adam with a beta_1 of 0.5. I think it is a key point. Can you provide some details about your optimizers. I suggest that we can use the setting in your paper: **All models were trained for 10 epochs, with the learning rate of 1e-4 until 5 epochs after which weight decay of 0.95 was applied per epoch.** Thanks for you attention
